resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -211,6 +228,11 @@ caws.download.thin_client=Downloading IDE thin client
 caws.environment.status=Dev Environment is currently: {0}
 caws.environment.view_pricing=Learn more about Dev Environment pricing
 caws.free.tier.subscription.storage=Other storage options are not currently available for your space. These require upgrading the billing tier.
+caws.getstarted.panel.description=Spend more time coding and less <br>time managing development <br>environments.
+caws.getstarted.panel.image=<img src="https://cdn.svgator.com/images/2022/11/Animated-GIF-best-practices.gif" height="400" width="500">
```

Comment:
let's delete this

Reply:
Is this description and gif deleting in every panels ?